app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -14,7 +14,10 @@
 
 @import 'partials/_exports.scss';
 
+
 $govuk-assets-path: '/customs-declare-exports/assets/';
 $govuk-images-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/images/";
 $govuk-fonts-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/fonts/";
-@import 'lib/govuk-frontend/govuk/all';
\ No newline at end of file
+@import 'lib/govuk-frontend/govuk/all';
+
+@import 'partials/_currency_input.scss';
```

Comment:
Is it required to be imported after `govuk/all` ? If not, then it mightt be better to keep it before. GDS had some rules to be the last imported part.

Reply:
Moved it to above govuk.  Turns out it didn't need to be last to work.